docs/development/extensions-contrib/aliyun-oss-extensions.md

```diff
@@ -22,33 +22,215 @@ title: "Aliyun OSS"
   ~ under the License.
   -->
 
+[Alibaba Cloud](https://www.aliyun.com) is the 3rd largest cloud infrastructure provider in the world. It provides its own storage solution known as OSS, [Object Storage Service](https://www.aliyun.com/product/oss).
+This document describes how to use OSS as Druid deep storage.
 
-To use this Apache Druid extension, make sure to [include](../../development/extensions.md#loading-extensions) `aliyun-oss-extensions` extension.
+## Installation
 
-## Deep Storage
+Use [pull-deps](../../operations/pull-deps.md) tool shipped with Druid to install this [extension](../../development/extensions.md#community-extensions) on middle manager and historical nodes.
```

Comment:
I might explicitly give the name of the extension here, since the link takes you to the section with all extensions. 
 
```suggestion
Use the [pull-deps](../../operations/pull-deps.md) tool shipped with Druid to install the `aliyun-oss-extensions` extension, as described [here](../../development/extensions.md#community-extensions) on middle manager and historical nodes.
```